blocks/src/checkout/class-nets-easy-checkout-block.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Nets Easy Checkout Block
+ *
+ * @package DIBS_Easy/Blocks
+ */
+
+use Automattic\WooCommerce\Blocks\Payments\Integrations\AbstractPaymentMethodType;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class Nets_Easy_Checkout_Block
+ */
+class Nets_Easy_Checkout_Block extends AbstractPaymentMethodType {
+	/**
+	 * The payment method name.
+	 *
+	 * @var string
+	 */
+	protected $name = 'nets_easy';
```

Comment:
This name does not match the gateway ID `dibs_easy`. Is that intentional?

https://github.com/krokedil/dibs-easy-for-woocommerce/blob/master/classes/class-nets-easy-gateway.php#L43

Reply:
Yes that is intentional. In this case, since we add multiple gateways, the name does not need to match the gateway id. This is only for the name that we use to [load the settings in the script later](https://github.com/krokedil/dibs-easy-for-woocommerce/blob/018e0dc564b2fa399fa3cf0568c63acedb59e418/blocks/src/checkout/index.tsx#L16) to then loop through the available gateways and create a blocks payment method for each of the used Nets gateways.